pepper-apis/src/main/resources/db-changes/schema/DDP-5277-revision-activity-naming-details.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="yufeng" id="20201112-add-i18n-activity-detail-table">
+        <createTable tableName="i18n_activity_detail">
+            <column name="i18n_activity_detail_id" type="bigint" autoIncrement="true" startWith="1">
+                <constraints primaryKey="true" primaryKeyName="i18n_activity_detail_pk"/>
+            </column>
+            <column name="study_activity_id" type="bigint">
+                <constraints nullable="false"
+                             references="study_activity(study_activity_id)"
+                             foreignKeyName="i18n_activity_detail_activity_fk"/>
+            </column>
+            <column name="language_code_id" type="bigint">
+                <constraints nullable="false"
+                             references="language_code(language_code_id)"
+                             foreignKeyName="i18n_activity_detail_language_fk"/>
+            </column>
+            <column name="revision_id" type="bigint">
+                <constraints nullable="false"
+                             references="revision(revision_id)"
+                             foreignKeyName="i18n_activity_detail_revision_fk"/>
+            </column>
+            <column name="name" type="varchar(500)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="second_name" type="varchar(500)"/>
+            <column name="title" type="varchar(1000)"/>
+            <column name="subtitle" type="varchar(500)"/>
+            <column name="description" type="varchar(500)"/>
+        </createTable>
+    </changeSet>
+
+    <changeSet author="yufeng" id="20201112-migrate-to-i18n-activity-detail">
+        <sql>
+            <comment>Insert row for each revision of each activity in each language.</comment>
+            insert into i18n_activity_detail
+                   (study_activity_id, language_code_id, revision_id, name, second_name, title, subtitle, description)
+            select i18n_act.study_activity_id,
+                   i18n_act.language_code_id,
+                   act_ver.revision_id,
+                   i18n_act.name,
+                   i18n_act.second_name,
+                   i18n_act.title,
+                   i18n_act.subtitle,
+                   i18n_act.description
+              from i18n_study_activity as i18n_act
+              join activity_version as act_ver on act_ver.study_activity_id = i18n_act.study_activity_id
+        </sql>
+        <dropTable tableName="i18n_study_activity"/>
+    </changeSet>
+
```

Comment:
Could we find a way to have these name, subtitle be instance of Template? Worries me that we have another table with translation text.

Reply:
We have other "translation text" stuff as well, such as "summaries". I think moving to Templates is a much bigger change. Can we tackle that separately?

Reply:
OK. I see just migrating existing stuff. Never mind.